backend/stakepoold/rpc/rpcserver/context.go

```diff
@@ -408,6 +408,32 @@ func (ctx *AppContext) UpdateUserDataFromMySQL() error {
 	return nil
 }
 
+// MyMasterPubKeys returns a map of master pubkeys to their account
+func (ctx *AppContext) MyMasterPubKeys() (map[string]string, error) {
+	acctMap, err := ctx.WalletConnection.ListAccounts()
+	if err != nil {
+		log.Errorf("MyMasterPubKeys: ListAccounts rpc failed: %v", err)
+		return nil, err
+	}
+	keyMap := make(map[string]string)
+	for acct := range acctMap {
+		// skip imported
+		if acct == "imported" {
+			continue
+		}
+		key, err := ctx.WalletConnection.GetMasterPubkey(acct, ctx.Params)
+		if err != nil {
+			log.Errorf("MyMasterPubKeys: GetMasterPubKey rpc failed: %v", err)
```

Comment:
I think this should probably be a fatal error which causes the RPC to return an error response immediately. I dont see any reason we should tolerate an error here. 

If there is something wrong with the way things are configured, the VSP operator should get a hard failure so they know they need to fix it straight away. The way this is currently written could make an error easy to miss

Reply:
Ok. That does make sense. It also makes me feel better about not checking the length of the map later.